test/test_msg.py

```diff
@@ -0,0 +1,35 @@
+import pynng
+from test._test_util import wait_pipe_len
+
+addr = 'tcp://127.0.0.1:42421'
+
+# timeout, ms
+to = 1000
+
+
+def test_socket_send_recv_msg():
+    with pynng.Pair0(listen=addr, recv_timeout=to) as s1, \
+            pynng.Pair0(dial=addr, recv_timeout=to) as s2:
+        wait_pipe_len(s1, 1)
+        pipe = s1.pipes[0]
+        msg = pipe.new_msg(b'oh hello friend')
+        assert isinstance(msg, pynng.Message)
+        assert msg.bytes == b'oh hello friend'
+        s1.sendmsg(msg)
```

Comment:
I'm slightly partial to `send_msg()` / `recv_msg()` personally.

Reply:
In my first version, I did use `send_msg` and `recv_msg`; I changed it to be more consistent with nng.  But you convinced me!  I'm going to change it back.